README.md

```diff
@@ -132,6 +132,7 @@ Limitations
   * `D`: Deleted file (not staged).
   * `M`: Modified file (not staged).
   * `?`: Untracked file.
+  * `!`: Ignored file.
```

Comment:
Could you please include a short description of the `--ignore` parameter to the README? I would put it into the [Usage](https://github.com/s3rvac/git-edit-index#usage) section.